From 9df1b4920089ed20da942f84ed6e2cc95657e162 Mon Sep 17 00:00:00 2001
From: Cheick Traore <cheick.traore-ext@st.com>
Date: Wed, 14 Aug 2024 10:19:47 +0200
Subject: [PATCH 1097/1695] dts: stm32: Change LPUART Clock on stm32mp257f-ev1
 board

Change LPUART clock from PLL5 to MSI_KER to use the wakeup feature of the
LPUART. MSI_KER is the only one clock ables to be used in low-power by
LPUART.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 52c727a29c6d9afa67c515a7211867b90e8f4dd5 dts: stm32: Change LPUART Clock on stm32mp257f-ev1 board]
Change-Id: I1f3bafd7fd476745c159ae2e50fe35dd97425a57
Signed-off-by: Cheick Traore <cheick.traore-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/398396
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi b/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
index b0199e5e5..db8827f6a 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
+++ b/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
@@ -73,7 +73,7 @@
 		FLEXGEN_CFG(36, XBAR_SRC_PLL5, 0, 3)
 		FLEXGEN_CFG(37, XBAR_SRC_PLL5, 0, 3)
 		FLEXGEN_CFG(38, XBAR_SRC_PLL5, 0, 3)
-		FLEXGEN_CFG(39, XBAR_SRC_PLL5, 0, 3)
+		FLEXGEN_CFG(39, XBAR_SRC_MSI_KER, 0, 0)
 		FLEXGEN_CFG(40, XBAR_SRC_LSE, 0, 0)
 		FLEXGEN_CFG(41, XBAR_SRC_PLL4, 0, 11)
 		FLEXGEN_CFG(42, XBAR_SRC_PLL7, 0, 16)
-- 
2.47.2

